### PR TITLE
fix: formatting to md from rst

### PR DESCRIPTION
**Pull request by TheSriram**
- DESIGN
- README
- DEVELOPMENT_GUIDE

*Issue #, if available:*

*Description of changes:*

*Checklist:*

- [ ] Write Design Document ([Do I need to write a design document?](https://github.com/awslabs/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.rst#design-document))
- [ ] Write unit tests
- [ ] Write/update functional tests
- [ ] Write/update integration tests
- [ ] `make pr` passes
- [ ] Write documentation

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+-   Validate SAM templates
+-   Get started with boilerplate Serverless Service in your chosen
+    Lambda Runtime `sam init`
+
+Get Started
+-----------
+
+Learn how to get started using the SAM CLI with these guides:
+
+-   [Installation](docs/installation.rst): Set up your macOS, Linux or
```
Comment (jfuss) [inline README.md]: Should we use this time to update these links to point to the aws docs we now have?

Comment (TheSriram) [reply README.md]: definitely. the installation docs already do, but need to move out the rest too.

Comment (jfuss) [reply README.md]: I wonder if we should just collapse the installation docs into the readme. Not sure if the extra hop is valuable. Or just link directly to the landing page which has the different installations: https://aws.amazon.com/serverless/sam/

Or link them here: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html but we need to update this one first to make pip an advanced install and default to installers first.

Comment (TheSriram) [reply README.md]: yeah moved it to the landing page, this way the first options seen to install are the official installation options.

**Pull request merged**